### DISCUSSION TITLE
Snapdragon 660,662,665

### DISCUSSION THREAD

**Discussion by Youtube0304** [skyline-emu/skyline#1527] [Q&A] [answered]
These intermediate processors are having a problem there starting gamer Cuphead, Hollow Knight Just show the FPS But it doesn't show the image in the game.Could you fix these problems please I beg 😓

**Comment by Allatel** [comment]
Thats not intermediate, thats low end. 730+ is intermediate.

**Comment by Crytonics** [comment, marked as answer]
You can't play with that cpu, because gpu have old drivers and can't render them.
But you can play Sonic mania, Celeste, and maybe Visual novels.

**Comment by ghost** [comment]
Too low but you can try 2d games they might run fine.
